fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java

```diff
@@ -145,54 +145,31 @@ public void testCreateView() throws SQLException, ClassNotFoundException {
     String sqlQuery = "select count(*) from warehouse as wr where wr.w_warehouse_sq_ft > 100";
     assertThat(getSize(sqlQuery)).isEqualTo(0);
 
-    String sql1 = "CREATE VIEW (name, description, cost, query, ds_set_id, destination_id, schema_name, table_name) "
-        + "values(\"warehouse_part\", \"Warehouse Partition\", 0, "
-        + "\"select * from canonical.public.warehouse as wr where wr.w_warehouse_sq_ft > 100\", "
-        + "1, 2, \"PUBLIC\", \"WAREHOUSE_PARTITION\")";
+    String sql1 = "CREATE VIEW warehouse_part set description = \"Warehouse Partition\", cost = 0, "
+        + "query=\"select * from canonical.public.warehouse as wr where wr.w_warehouse_sq_ft > 100\", "
+        + "destination_id=2, schema_name=\"PUBLIC\", table_name = \"WAREHOUSE_PARTITION\"";
     Connection connection = DriverManager.getConnection("jdbc:quark:fat:db:", props);
     connection.createStatement().executeUpdate(sql1);
     connection.close();
 
     assertThat(getSize(sqlQuery)).isEqualTo(1);
   }
 
-  @Test(expected = SQLException.class)
-  public void testCreateWrongParam() throws SQLException, ClassNotFoundException {
```

Comment:
It is actually insufficient number of params here. table_name has not been specified in this test. Other test case is with regards to altering View. We only allow `id = <constant>` in  where clause currently for alter views/datasources.